server/src/serializers/ReviewSerializer.js

```diff
@@ -5,11 +5,10 @@ class ReviewSerializer {
     for (const attribute of allowedAttributes) {
       serializedReview[attribute] = review[attribute]
     }
-
     serializedReview.voteCount = await review.$voteCount()
-  
+    serializedReview.votes = await review.$relatedQuery("votes")
```

Comment:
per the feedback I gave in your react component, you wont be sending back all the votes, just the current user's vote value. This means that in the router, you'll need to pass the currentUser in to your PlanetSerializer and down to here